### PR TITLE
Feat/wt 2155 wallet connect bridge

### PR DESCRIPTION
**Pull request by ZacharyCouchman**
Adding in WalletConnect options to the bridge for connections.

- [ ] `breaking-change:` if you have introduced modification that necessitates immediate adjustments by this SDK's users to their applications, clients, or integrations to avert disruptions to existing features or functionalities.
- [x] `feat:`, `fix:`, `refactor:`, `docs:`, or `chore:`.

# Summary

# Detail and impact of the change

## Added 

WalletConnect options into the Bridge Widget

## Changed

How the connect callback is passed into the useWalletConnect hook.

### CODE REVIEW

File: packages/checkout/widgets-lib/src/lib/hooks/useWalletConnect.ts [hunk 1]
```diff
@@ -1,49 +1,95 @@
...
   const [walletConnectModal, setWalletConnectModal] = useState<WalletConnectModal | null>(null);
-  // const [displayUri, setDisplayUri] = useState<string | null>(null);
+  const displayUri = useRef<string>('');
+
+  useEffect(() => {
+    if (!checkout) return;
+    (async () => {
+      // const connectConfig = await checkout.config.remote.getConfig('connect');
+      setIsWalletConnectEnabled(
+        /** connectConfig?.walletConnect */ true
```
Comment (sharonsheah) [inline packages/checkout/widgets-lib/src/lib/hooks/useWalletConnect.ts:30]: We probably want to add these back in.

**Review by sharonsheah — APPROVED** [review]
LGTM! 🚀

File: packages/checkout/widgets-sample-app/src/components/ui/marketplace-orchestrator/MainPage.tsx [hunk 2]
```diff
@@ -120,7 +134,25 @@ export const MainPage = () => {
     bridgeWidget.update(languageUpdate);
     swapWidget.update(languageUpdate);
     onRampWidget.update(languageUpdate);
-  }, [onRampWidget, web3Provider])
+  }, [onRampWidget, web3Provider]);
+
+  // When widgets are already mounted, the theme only changes once
+  // When the widgets are mounted, the WalletConnect theme is fixed to the theme before widget is mounted
+  const toggleTheme = useCallback(() => {
```
Comment (dreamoftrees) [inline packages/checkout/widgets-sample-app/src/components/ui/marketplace-orchestrator/MainPage.tsx:141]: Nice update!

**Review by dreamoftrees — APPROVED** [review]
👍 Nice work!

**Pull request merged**